react/src/App.js

```diff
@@ -53,13 +53,15 @@ function App(props) {
                                         <Route path="/deal-transfers" element={<DealTransfersPage />} />
                                         <Route path="/mpool" element={<MpoolPage />} />
                                         <Route path="/settings" element={<SettingsPage />} />
-                                        <Route path="/piece-directory" element={<PieceDirectoryPage />} />
                                         <Route path="/deals/:dealID" element={<DealDetail />} />
                                         <Route path="/legacy-deals/:dealID" element={<LegacyDealDetail />} />
-                                        <Route path="/inspect" element={<InspectPage />} />
+                                        <Route path="/lid" element={<LIDPage />} />
+                                        <Route path="/sectors-list" element={<SectorsListPage />} />
+                                        <Route path="/sectors-list/from/:cursor/page/:pageNum" element={<SectorsListPage />} />
+                                        <Route path="/piece-doctor" element={<PieceDoctorPage />} />
```

Comment:
Both`/piece-doctor` and `/inspect` route to the PieceDoctor, I suggest we unify them somehow, as this is a bit confusing.